packages/store/src/doc-providers.ts

```diff
@@ -48,16 +48,8 @@ const signaling = isLocalhost ? LOCAL_SIGNALING : DEFAULT_SIGNALING;
 export class DebugDocProvider extends WebrtcProvider implements DocProvider {
   constructor(room: string, doc: Y.Doc, options?: { awareness?: Awareness }) {
     super(room, doc, {
-      awareness: options?.awareness ?? null,
+      awareness: options?.awareness ?? undefined,
```

Comment:
Personally I prefer explicitly passing in `null`, but here `options?.awareness` without `??` looks adequate.

Reply:
the type of y-webrtc has changed which disallow passing a `null` here